docs/topics_5/templates/document-attributes.adoc

```diff
@@ -35,7 +35,7 @@
 // * Doc metadata *
 // ****************
 :DocInfoProductName: Migration Toolkit for Applications
-:DocInfoProductNumber: 5.0
+:DocInfoProductNumber: 5.0.1
 :DocInfoProductNameURL: migration_toolkit_for_applications
```

Comment:
Not sure this is a good change. Changing from 2 to 3 digits is problematic. If you were changing `5.0.0` to `5.0.1`, I would have no problem with that. You need to check how this attribute is used in the docs.

Reply:
The only place it appears in text users see is in the master.adoc of release-notes-5.0. it also appears in the master-docinfo.xml files. If we do not make this change, then the opening sentence of the release notes will refer to version 5.0, not 5.0.1. If that's OK, I'll remove the change.

Reply:
Release notes are for y versions, not individual z versions. So remove this change.

Reply:
Removed.